authentication-action-archetype/pom.xml

```diff
@@ -0,0 +1,16 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.curity</groupId>
+        <artifactId>identityserver.plugins.archetypes</artifactId>
+        <version>1.3.0</version>
```

Comment:
Can you bump this so 1.3.1 as I had to do that on the master branch to push to Central